### PR TITLE
getting-started: add arch linux instructions

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Tested on my own machine

### CODE REVIEW

File: content/getting-started/linux.md [hunk 1]
```diff
@@ -122,6 +122,9 @@ sudo apt-get update
 sudo apt-get install clang-8 llvm-8-dev lld-8 libclang-8-dev
 ```
 
+For Arch Linux, install [the llvm package](https://www.archlinux.org/packages/extra/x86_64/llvm/).
+It must be version 8.0.0 or above.
```
Comment (aykevl) [inline content/getting-started/linux.md]: > It must be version 8.0.0 or above.

But it must be LLVM 8 (8.1.0 would be OK, if that gets released).

Comment (johanbrandhorst) [reply content/getting-started/linux.md]: I wanted to illustrate anything => 8.0.0, is the strict requirement  8.0.0 <= x < 9.0.0?

**Comment by johanbrandhorst** [comment]
Wait, stop, dont' merge this, the archi linux package is missing wasm-ld.

**Comment by johanbrandhorst** [comment]
I'll do some more investigating

**Comment by johanbrandhorst** [comment]
OK so we also need the `lld` package, but disappointingly it seems `tinygo` looks for `wasm-ld-8` and can't find `wasm-ld`. I'll raise a separate issue about that.

**Comment by johanbrandhorst** [comment]
Raise https://github.com/tinygo-org/tinygo/issues/305, but the solution is not obvious to me yet. Feedback appreciated. These instructions can go in as they are now that I've added the symlink instructions.

**Comment by johanbrandhorst** [comment]
I've updated this based on https://github.com/tinygo-org/tinygo/pull/323 being merged. Do you have anymore comments @aykevl?

File: content/getting-started/linux.md (hunk 1 above)
Comment (aykevl) [reply content/getting-started/linux.md]: Oops, never replied to this.
Yes, the current requirement is on 8.x.x, so `8.0.0 <= x < 9.0.0`.

**Comment by aykevl** [comment]
> Do you have anymore comments

Yes, see above.

File: content/getting-started/linux.md (hunk 1 above)
Comment (johanbrandhorst) [reply content/getting-started/linux.md]: Updated

File: content/getting-started/linux.md [hunk 2]
```diff
@@ -122,6 +122,10 @@ sudo apt-get update
 sudo apt-get install clang-8 llvm-8-dev lld-8 libclang-8-dev
 ```
 
+For Arch Linux, install the [llvm](https://www.archlinux.org/packages/extra/x86_64/llvm/)
+and [ldd](https://www.archlinux.org/packages/extra/x86_64/ldd/) packages.
```
Comment (aykevl) [inline content/getting-started/linux.md]: Reading it again, you only mention llvm and lld. Clang is also a dependency, or will it be installed along with one of those packages?

Comment (johanbrandhorst) [reply content/getting-started/linux.md]: Great question, I thought it was a dependency but I might just list it explicitly. Will update!

Comment (johanbrandhorst) [reply content/getting-started/linux.md]: Updated

**Comment by aykevl** [comment]
Thank you!

**Pull request merged**